.github/workflows/prettier.yml

```diff
@@ -0,0 +1,31 @@
+name: Prettier Lint
+on: 
+  push:
+    branches:
+      - dev
+
+jobs:
+  prettier-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+            node-version: 18
+
+      - name: Install dependencies
+        run: |
+          yarn add prettier
+
+      - name: Run Prettier
+        run: yarn format
+
+      - name: Commit and push changes
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: Apply Prettier changes
+          branch: ${{ github.head_ref }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I suggest using a more descriptive name so we know what this "GITHUB_TOKEN" is for, as we have a few other repo secrets

Reply:
It's automatically generated by GitHub Actions.